test/io/fixtures.yaml

```
@@ -196,6 +196,24 @@ roleclaims:
           - obj_key: obj_value
     expected_status: 401 # fails because it compares an object with a string
 
+jwtAudRoleclaims:
```

```suggestion
jwtaudroleclaims:
```

Also not sure if it's convention or not, but for consistency with the other names let's keep it in lower case.